docs/guides/how-to-round-robin-load-balance-tunnels-with-ngrok-cloud-edges.mdx

```diff
@@ -0,0 +1,305 @@
+  <TabItem value="cli" label="ngrok CLI" default>
+
+```bash
+ngrok api reserved-domains create \
+	--domain ${NGROK_SUBDOMAIN}.ngrok.app
+```
+
+- Replace or set `NGROK_SUBDOMAIN` as the subdomain you'd like to use for this guide.
+
+
+  </TabItem>
+  <TabItem value="curl" label="CURL">
+
+```bash
+curl \
+	-X POST https://api.ngrok.com/reserved_domains \
+	-H "Authorization: Bearer ${NGROK_API_KEY}" \
+	-H "Content-Type: application/json" \
+	-H "Ngrok-Version: 2" \
+	-d @- <<BODY
+	{
+		"name":"${NGROK_SUBDOMAIN}.ngrok.app",
+	}
+BODY
+```
+
+- Replace or set `NGROK_API_KEY` to your ngrok API key.
+- Replace or set `NGROK_SUBDOMAIN` as the subdomain you'd like to use for this guide.
+
+
+  </TabItem>
+</Tabs>
+
+After running, you should see the following:
+
+```json
+200 OK
+{
+   "id":"rd_2MT5Bqt0UzU0mFQ0zr8m1UQWCfm",
+   ...
+}
+```
+
+When you have completed this step, you can move on to the next step.
+
+## **Step 2** — Create a ngrok Cloud Edge
+
+Now we can create a ngrok Cloud Edge that points to our 
+newly reserved domain on port `443`:
+
+<Tabs groupId="guide">
+  <TabItem value="cli" label="ngrok CLI" default>
+
+```bash
+ngrok api reserved-domains create \
+	--domain ${NGROK_SUBDOMAIN}.ngrok.app
+```
+
+- Replace or set `NGROK_SUBDOMAIN` with the value used in the previous step.
+
+
+  </TabItem>
+  <TabItem value="curl" label="CURL">
+
+```bash
+curl \
+	-X POST https://api.ngrok.com/edges/https \
+	-H "Authorization: Bearer ${NGROK_API_KEY}" \
+	-H "Content-Type: application/json" \
+	-H "Ngrok-Version: 2" \
+	-d @- <<BODY
+	{
+		"description": "",
+		"metadata": "{}",
+		"hostports": ["${NGROK_SUBDOMAIN}.ngrok.app:443"]
+	}
+BODY
+```
+
+- Replace or set `NGROK_API_KEY` to your ngrok API key.
+- Replace or set `NGROK_SUBDOMAIN` with the value used in the previous step.
+
+
+  </TabItem>
+</Tabs>
+
+After running, you should see:
+
+```json
+{
+  "id": "edghts_2MT9nzunlKgQ0KRfmA5QK5iiI1J",
+  ...
+  "hostports": [
+    "${NGROK_SUBDOMAIN}.ngrok.app:443"
+  ],
+  ...
+}
+```
+
+Export the `id` value as a new environment variable named `NGROK_EDGE_ID`:
+
+```bash
+export NGROK_EDGE_ID=your-edge-id
+```
+
+Now that we have an edge, we can move onto the next step.
+
+## **Step 3** — Create a Tunnel Group Backend
+
+Now that we have an edge we can begin to build the parts to load balance. 
+Lets create a tunnel-group backend to do traffic management on our 
+edge with a label named `region=us`:
+
+<Tabs groupId="guide">
+  <TabItem value="cli" label="ngrok CLI" default>
+
+```bash
+ngrok api backends tunnel-group create \
+	--labels region=us \
+	--description "us tunnel group"
+```
+
+  </TabItem>
+  <TabItem value="curl" label="CURL">
+
+```bash
+curl \
+	-X POST https://api.ngrok.com/backends/tunnel_group \
+	-H "Authorization: Bearer ${NGROK_API_KEY}" \
+	-H "Content-Type: application/json" \
+	-H "Ngrok-Version: 2" \
+	-d @- <<BODY
+	{
+		"description": "us tunnel group",
+		"metadata": "{}",
+		"labels": {
+				"region": "us"
```

Comment:
this feels a little confusing just because region has a special meaning for ngrok. should we use a different label for this example?

Reply:
We could. The next guide (weighted) builds on this and introduces `region=eu` in addition to this one.

Reply:
Would also have to change the graphic.

Reply:
ah, ok, it's fine for now. thanks!